README.md

```diff
@@ -10,11 +10,11 @@ Typescript wrapper for [supranational/blst](https://github.com/supranational/bls
 
 | OS / Arch     | binary name | Node                               |
 | ------------- | ----------- | ---------------------------------- |
-| Linux / x64   | linux-x64   | 14, 16, 17, 18, 20 |
-| Linux / arm64 | linux-arm64 | 14, 16, 17, 18, 20 |
-| Windows / x64 | win32-x64   | 14, 16, 17, 18, 20 |
-| OSX / x64     | darwin-x64  | 14, 16, 17, 18, 20 |
-| OSX / M1      | -\*         | -\*                                |
+| Linux / x64   | linux-x64   | 16, 17, 18, 20 |
+| Linux / arm64 | linux-arm64 | 14, 16, 18, 20 |
+| Windows / x64 | win32-x64   | 16, 17, 18, 20 |
+| macOS / x64     | darwin-x64  | 16, 17, 18, 20 |
+| macOS / arm64      | darwin-arm64  | 16, 17, 18, 20 |
```

Comment:
Generally we have a policy to only support versions that are under active maintenance.  16 went off in Sept 2023 and 17 is not a supported version according to nodes table.  I think we should remove those as they could have security concerns that do not get patched by node anymore.

We should also consider added 21 to the testing matrix and to the description above.

https://nodejs.org/en/about/previous-releases

Reply:
btw good catch up updating this @jeluard!!! thanks!!

Reply:
This PR is about aligning docs with what exists for now. I'd rather open a new PR to update the versions (i.e. adding `21` might require other changes)

Reply:
sounds good for `21`.  💯

Reply:
we should remove the old versions though from the matrix of "supported"

Reply:
:+1: